### ISSUE TITLE
Division-method is invalid for PLAYER entity

### ISSUE THREAD

**Issue by francoispope** [diecode/KillerMoney#15]
When setting `Division-method: 'SHARED'` for the player entity, the console spits out a null error.
```
[19:01:16 WARN]: [KillerMoney] Invalid "Division-method" option at PLAYER in fram-realm section: null. Using default value: "LAST_HIT"
```
`entities.yml` is set as follows, which matches your docs.
```
PLAYER:
    'fram-realm':
        Money:
            Enabled: true
            Chance: 10%
            Value: 50
            Division-method: 'LAST_HIT'
        Custom-commands:
            Enabled: true
            Run-method: 'ALL'
            Commands:
                '1':
                    Command: 'broadcast §e{player} §4murdered §7for money!'
                    Chance: 5%
        Custom-item-drop:
            Enabled: false
            Keep-default-items: false
            Items:
                '1':
                    ItemStack:
                        ==: org.bukkit.inventory.ItemStack
                        type: GOLD_NUGGET
                    Random-amount: 1 ? 5
                    Chance: 100%
        Cash-transfer:
            Enabled: true
            Percent: 10%
            Max-amount: 500
            Chance: 100%
            Division-method: 'SHARED'
    'fram-realm_nether':
        Money:
            Enabled: true
            Chance: 10%
            Value: 50
            Division-method: 'LAST_HIT'
        Custom-commands:
            Enabled: true
            Run-method: 'ALL'
            Commands:
                '1':
                    Command: 'broadcast §7They say rules don''t apply in the §4underworld§7...'
                    Chance: 5%
        Custom-item-drop:
            Enabled: false
            Keep-default-items: false
            Items:
                '1':
                    ItemStack:
                        ==: org.bukkit.inventory.ItemStack
                        type: GOLD_NUGGET
                    Random-amount: 1 ? 5
                    Chance: 100%
        Cash-transfer:
            Enabled: true
            Percent: 33%
            Max-amount: 5000
            Chance: 100%
            Division-method: 'SHARED'
```

**Comment by kgahor** [comment]
Im investigating

**Comment by kgahor** [comment]
It's fixed, It will be available in 4.2.0 soon.
